draft-ietf-quic-http.md

```diff
@@ -1025,41 +1029,39 @@ RST bit set, as a stream error of type HTTP_CONNECT_ERROR
 
-## Request Prioritization {#priority}
+## Prioritization {#priority}
 
 HTTP/3 uses a priority scheme similar to that described in {{!RFC7540}}, Section
-5.3. In this priority scheme, a given stream can be designated as dependent upon
-another request, which expresses the preference that the latter stream (the
-"parent" request) be allocated resources before the former stream (the
-"dependent" request). Taken together, the dependencies across all requests in a
-connection form a dependency tree.
-
-When a client request is first sent, its parent and weight are determined by the
-PRIORITY frame (see {{frame-priority}}) which begins the stream, if present.
-Otherwise, the element is dependent on the root of the priority tree.
-Placeholders are also dependent on the root of the priority tree when first
-allocated.  Pushed streams are initially dependent on the client request on
-which the PUSH_PROMISE frame was sent. In all cases, elements are assigned an
-initial weight of 16 unless a PRIORITY frame begins the stream.
-
-The structure of the dependency tree changes as PRIORITY frames on the control
-stream modify the dependency links between requests. The PRIORITY frame
-{{frame-priority}} identifies a prioritized element. The elements which can be
-prioritized are:
+5.3. In this priority scheme, a given element can be designated as dependent
+upon another element. This information is expressed in the PRIORITY frame
+{{frame-priority}} which identifies the element and the dependency. The elements
+that can be prioritized are:
 
 - Requests, identified by the ID of the request stream
 - Pushes, identified by the Push ID of the promised resource
   ({{frame-push-promise}})
 - Placeholders, identified by a Placeholder ID
 
-An element can depend on another element or on the root of the tree.  A
-reference to an element which is no longer in the tree is treated as a reference
-to the root of the tree.
+Taken together, the dependencies across all prioritized elements in a connection
+form a dependency tree. An element can depend on another element or on the root
+of the tree. A reference to an element which is no longer in the tree is treated
+as a reference to the root of the tree. The structure of the dependency tree
+changes as PRIORITY frames modify the dependency links between prioritized
+elements.
 
 Due to reordering between streams, an element can also be prioritized which is
 not yet in the tree. Such elements are added to the tree with the requested
 priority.
 
+When a prioritized element is first created it has a default inital weight
+of 16 and a default dependency. Requests and placeholders are dependent on the
+root of the priority tree, pushes are dependent on the client request on which
+the PUSH_PROMISE frame was sent.
+
+Requests may override the default intial values by including a PRIORTIY frame
+(see {{frame-priority}}) at the beginning of the stream. Pushes and placeholders
```

Comment:
```suggestion
(see {{frame-priority}}) at the beginning of the stream. These priorities
```
It's not just pushes and placeholders -- this statement is true for any of them.